app/controllers/v1/fake_providers_controller.rb

```diff
@@ -0,0 +1,27 @@
+module V1
+  class FakeProvidersController < ApiController
+    def show
+      render json: provider_details
+    end
+
+    private
+
+    def provider_details
+      {
+        providerOffices: Array.new(rand(1..3)).map { provider_office },
+        contactId: rand(1..100_000),
+        contactName: params[:username]
+      }
+    end
+
+    def provider_office
+      {
+        providerfirmId: rand(1..100_000),
+        officeId: rand(1..100_000),
```

Comment:
Actually, the `providerfirmId` will be the same for every office.  I'm talking to Asrar about getting the structure changed so that the `providerfirmId` is outside the the `providerOffices` array, but until such time as it is, we should really ensure that the same id appears in every occurance of `providerOffices`